src/definitions/z.json

```diff
@@ -0,0 +1,37 @@
+[
+  {
+    "display": "z-index",
+    "description": "Specifies the stack order of an element",
+    "link": "https://www.w3schools.com/cssref/pr_pos_z-index.asp",
+    "propertyName": "z-index",
+    "valuetemplate": "{0}",
+    "categories": ["container"],
+    "tags": "container",
+    "controls": [
+      {
+        "type": "select",
+        "default": "auto",
+        "label": "Value",
+        "options": [
+          {
+            "name": "Auto",
+            "value": "auto"
+          },
+          {
+            "name": "Number",
```

Comment:
When some select number from the list, The Ui is supposed to show a number box for customer value.
Please refer `column-count`  (or similar) for referrence.

Reply:
I'm not quite grasping it.  It is pretty much applying `controls`?
```
          {
            "name": "Custom",
            "value": {
              "controls": [
                {
                  "type": "number",
                  "label": "Count"
                }
              ]
            }
          }
```

Reply:
yes, it is